### PR TITLE
Editor

### PR DESCRIPTION
**Pull request by sadathshariff**
Added Text Editor From React Quil

### CODE REVIEW

File: src/components/AddNote/AddNote.jsx [hunk 1]
```diff
@@ -104,17 +121,26 @@ export const AddNote = () => {
                 value={note.notes}
                 onChange={(e) => handleChange(e)}
                 required
-              />
+              /> */}
```
Comment (rdshinde) [inline src/components/AddNote/AddNote.jsx:124]: Remove this commented code.

File: src/components/AddNote/AddNote.jsx [hunk 2]
```diff
@@ -104,17 +121,26 @@ export const AddNote = () => {
...
             <div className="addNote-footer">
               <Button
                 name={"Cancel"}
                 btnclass={"btn-danger"}
                 onClick={handleClose}
               />
-              <Button name={"Add Note"} btnclass={"btn-primary"} />
+              {!isEditing && (
+                <Button name={"Add Note"} btnclass={"btn-primary"} />
+              )}
```
Comment (rdshinde) [inline src/components/AddNote/AddNote.jsx:134]: I liked that your breaking small components also.

File: src/context/Note/context.js [hunk 3]
```diff
@@ -52,8 +53,7 @@ const NoteProvider = ({ children }) => {
         {
           headers: { authorization: token },
         }
-      );
-      console.log("Updated Res", res);
+      );      
       setUserNotes(res.data.notes);
     } catch (error) {
       console.error("Update Notes Error", error);
```
Comment (rdshinde) [inline src/context/Note/context.js:59]: If you are using toast for messages, then this console.log can be moved to toast.

**Review by rdshinde — APPROVED** [review]
Small suggestions. Otherwise looks good to merge.

**Review by nishasen — APPROVED** [review]
Your code looks clean and good, liked how you are using editor

File: src/context/Note/context.js (hunk 3 above)
Comment (sadathshariff) [reply src/context/Note/context.js:59]: Yeah will do that

**Pull request merged**